core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -525,4 +526,45 @@ private User getBuddyUser(BuddyDTO buddy)
 		}
 		return Optional.empty();
 	}
+
+	@Transactional
+	void broadcastBuddyInfoChangedMessageIfNeeded(User updatedUserEntity, UserDTO originalUser)
```

Comment:
Please rename into ``broadcastUserInfoChangeToBuddies``. Why not always broadcast the event? That way, we immediately have the full feature of informing buddies about changes in the personal info of their friends.